### PR TITLE
Added diagrams for IBM Provider

### PR DESCRIPTION
**Pull request by MalarvizhiK**
Added diagrams for IBM Provider

### CODE REVIEW

**Comment by MalarvizhiK** [comment]
@mingrammer Could you please review PR ?

File: diagrams/ibm/security.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# This module is automatically generated by autogen.sh. DO NOT EDIT.
```
Comment (mingrammer) [inline diagrams/ibm/security.py:1]: Aren't there any security nodes :) ?

Comment (MalarvizhiK) [reply diagrams/ibm/security.py:1]: @mingrammer Please review. I added more diagrams.

Comment (MalarvizhiK) [reply diagrams/ibm/security.py:1]: @mingrammer can you approve this week end ?

**Review by gabriel-tessier — COMMENTED** [review]
The name of the files in network may not be correct, better to put lower case and "-" so the generated name will have camel case.
for ex: 
class Directlink(_Network):
    _icon = "DirectLink.png"
    
if the name of the file is direct-link.png  the class generated will be DirectLink which look  better.

Also just a global remarks there's no alias I think you can at least add FloatingIP after renaming the file floating-ip.png ! Same for all the class with VPN and VPC.

**Comment by MalarvizhiK** [comment]
Review comments noted. I am working on it.

**Comment by MalarvizhiK** [comment]
@gabriel-tessier Please review now. I have corrected as per review comments.

**Comment by mingrammer** [comment]
I’ll review it tomorrow!

**Comment by MalarvizhiK** [comment]
@mingrammer @gabriel-tessier Can you review this weekend ?

**Review by mingrammer — APPROVED** [review]
LGTM! Thank you. Sorry for the late review.

**Comment by mingrammer** [comment]
v0.20.0 is now released with IBM cloud support.

**Comment by MalarvizhiK** [comment]
Thankyou @mingrammer

**Pull request merged**